src/js/components/Search/OrDelimiter.tsx

```diff
@@ -0,0 +1,19 @@
+import { type CSSProperties, memo } from 'react';
+import { Flex } from 'antd';
+import { useTranslationFn } from '@/hooks';
+
+const lineStyle: CSSProperties = { width: 1, flex: 1, backgroundColor: '#f0f0f0' };
+
+const OrDelimiter = memo(() => {
+  const t = useTranslationFn();
+  return (
+    <Flex vertical={true} gap={8} align="center">
+      <div style={lineStyle}></div>
+      <div style={{ fontWeight: 'bold', color: '#595959' }}>{t('OR')}</div>
+      <div style={lineStyle}></div>
```

Comment:
Might be useful to use a divider with the vertical option instead of divs. If not the divs can be self closing. `<div ..../>`

Reply:
ant has too many styles, ive factored this into a mini component tho